HEC.FDA.ModelTest/unittests/PairedDataShould.cs

```diff
@@ -89,14 +89,19 @@ public void Multiply()
         {
             double[] multiplierXs = { 2, 3, 4 };
             double[] multiplierYs = { .5, .5, .5 };
-            double[] expectedXs = { 1, 1.999, 2, 3, 4, 4.001, 5 };
-            double[] expectedYs = { 0, 0, 1, 1.5, 2, 4.001, 5 };
+
+            double[] expectedXs = { 1, 2, 3, 4, 5 };
+            double[] expectedYs = { 0, 1, 1.5, 2, 5 };
             PairedData multiplier = new PairedData(multiplierXs, multiplierYs);
```

Comment:
I'd like to see 3 test cases: one where the multiplier has fewer points than the base, one where it has more, and one where the coordinate count matches.

Reply:
great idea